ed448-goldilocks/src/sign/expanded.rs

```diff
@@ -61,7 +64,7 @@ impl ExpandedSecretKey {
 
         let point = EdwardsPoint::GENERATOR * scalar;
         let public_key = VerifyingKey {
-            compressed: point.compress(),
+            compressed: point.to_affine().compress(),
```

Comment:
Couldn't remove this because we are storing `CompressedEdwardsY` in `VerifyingKey` and `GroupEncoding` doesn't return that.

I'm happy to change the type.
I'm planning on removing the type entirely anyway to align with the other curves in this repo, so we can wait for that as well.